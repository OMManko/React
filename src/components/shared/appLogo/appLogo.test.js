import React from 'react';
import { shallow } from 'enzyme';
import AppLogo from "../appLogo/appLogo";

let component = shallow(<AppLogo/>);

describe('<AppLogo />', () => {
    beforeEach(() => {
        component = shallow(<AppLogo/>);
    });
    it('should render logo and match snapshot', () => {
        expect(component).toMatchSnapshot();
    });
});
