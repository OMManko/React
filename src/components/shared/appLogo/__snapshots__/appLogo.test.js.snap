// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppLogo /> should render logo and match snapshot 1`] = `ShallowWrapper {}`;
